ui/src/layouts/chat.rs

```diff
@@ -29,14 +29,15 @@ pub fn ChatLayout(cx: Scope<Props>) -> Element {
             ChatSidebar {
                 route_info: cx.props.route_info.clone()
             },
-            (state.read().ui.is_minimal_view() && state.read().ui.sidebar_hidden || !state.read().ui.is_minimal_view()).then(|| rsx!(
+            // todo: figure out why this isn't working, fix, and  uncomment. need a special view for mobile devices.
```

Comment:
this works on my end, I can try to replicate an issue.

Reply:
this says that if  not ((mobile view and no sidebar) or (not mobile view) ) then don't render anything...

Reply:
The sidebar would be rendered since it's not hidden and minimal_view is true. Looked into this a bit more and the welcome page needs a back button and we need a resize handler on the app. Noted an issue @sdwoodbury was having on linux that i'm not able to replicate but did see.